Dockerfile

```diff
@@ -1,20 +1,18 @@
-FROM broadinstitute/scala-baseimage
+FROM openjdk:8
+
+# To run, must build the jar using ./docker/build.sh
 
 # Rawls' default port
 EXPOSE 8080
 EXPOSE 5050
 
 ENV GIT_MODEL_HASH $GIT_MODEL_HASH
 
-ADD . /rawls
-
-# catch sbt issues separately
-RUN cd /rawls && sbt update && echo "sbt updated successfully."
-
-RUN ["/bin/bash", "-c", "/rawls/docker/install.sh /rawls"]
+RUN mkdir /rawls
+COPY ./rawls-assembly-0.1.jar /rawls
 
 # Add Rawls as a service (it will start when the container starts)
-RUN mkdir /etc/service/rawls
+RUN mkdir -p /etc/service/rawls
 ADD docker/run.sh /etc/service/rawls/run
```

Comment:
This and the previous line are really artifacts of how the previous base image worked and don't make sense here. I would remove them, along with `docker/run.sh`, and add a `CMD` directive. I think `CMD` is more appropriate than `ENTRYPOINT` for this case.